src/context/CartContext.tsx

```diff
@@ -1,13 +1,16 @@
+/* eslint-disable @typescript-eslint/no-unused-vars */
```

Comment:
any reason why we disable this same with below

Reply:
Type CartContextType and CartContext are expecting the product and id inputs to be used inside the function, but this is not necessary with Context